bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs

```diff
@@ -1,16 +1,18 @@
         readonly string plaintext = "{\"object_id\":\"some-external-id-string-123\",\"style_id\":\"jefferson-county-ballot-style\",\"contests\":[{\"object_id\":\"justice-supreme-court\",\"ballot_selections\":[{\"object_id\":\"john-adams-selection\",\"vote\":1},{\"object_id\":\"write-in-selection\",\"vote\":1,\"extra_data\":\"some writein value\"}]}]}";
         [Test]
+        [Ignore("Memory Access Violation")]
```

Comment:
what's this about?

Reply:
These tests have a Memory Access Violation issue. This is an issue for this:
https://github.com/microsoft/electionguard-cpp/issues/193